Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN echo -e "keyserver-options auto-key-retrieve" >> /etc/pacman.d/gnupg/gpg.con
 RUN echo -e "#!/bin/bash\nif [[ \"$1\" == \"--version\" ]]; then echo 'fake 244 version'; fi\nmkdir -p /var/cache/pikaur\n" >> /usr/bin/systemd-run && \
     chmod +x /usr/bin/systemd-run
 
+COPY rootfs/etc/makepkg.conf /etc/makepkg.conf
```

Comment:
What is exactly changed from the default? Can it be just a `sed` over the specific line on `makepkg.conf`? My concern is upstream changes can go unnoticed in other areas if we freeze the file in our repo. Might be better to search/replace that specific line.

Reply:
I have now included a check that will fail the docker container construction if the upstream makepkg.conf gets changed so it won't go unnoticed.

As for why it is not a simple sed command, well this way the file is in the repo for reference and the process of changing something is straighforward. Moreover, speaking of upstream changes, with sed you will never know if you substituted more than what was intended to just because of upstream change that went unnoticed.

Reply:
The checksum check additional step refers to a /etc/makepkg.conf file that was calculated from a clean archlinux:base-devel docker container. Once "!check" is restored to "check" the hash will match proving That was the only modification that was done to the one you are currently using.